### ISSUE TITLE
Unify imports at the top of a file

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot-sdk#2081]
I often see this in some runtime files, where the imports are spread over multiple blocks. The criteria to how these blocks were formed or the logic behind it is not documented and makes it impossible for `cargo fmt` to remove duplicated or for the developer to find things:  

https://github.com/paritytech/polkadot-sdk/blob/f6560c2b7226ea756ade18df42018c3eaf3be2e0/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L46-L59

I think it would be smart to not split the imports with a newline, but have them all in one block.  
The only exception would be feature gated imports. As structure i propose:

```rust
// Normal imports:
use ...;
use ...;
...

// Feature gated imports:
#[cfg(feature = "std")]
use ::{
    // Std imports all in one block
};
```

Note that this will cause huge merge conflicts with existing MRs, so please check what MRs are affected first.

**Comment by altonen** [comment]
On the client side, in some files there is a pattern where crate, external and std imports are in their own blocks. I think that's preferable to clumping everything in one block but since there's no guideline, it's not as consistent as it could perhaps be.

**Comment by bkchr** [comment]
Yeah, we should come up with some kind of consistent way for doing this and then putting it into the `CONTRIBUTING.md`. Personally I think moving all to one block or having blocks split in the way @altonen said is fine to me.
